README.md

```diff
@@ -308,6 +308,127 @@ for vehicle in vehicle_path_list:
 
 All of the simulations will be launched in one go. The user can check the status of the simulations and retrieve the results when they are ready. Check the FAQ section for more information on how to do this.
 
+## Task management
+
+
+### Using the `Task` class
+
+As you may have noticed, both the sync and async runs of the scenarios show above return a `Task` object to the user.
+This object provides several methods to manage a task submitted to the **Inductiva API**.
+For instance, you can get information about the task (its current status, when it started, information on the machine it is running, ...), kill it while
+it is still running if you've changed your mind, or download its output files (all of them or only those you need) after it completed.
+
+Check out some examples after submitting an asynchronous simulation:
+
+```python
+# `scenario`` constructed as in the examples above.
+# ... stands for arguments related to each specific scenario.
+task = scenario.simulate(..., run_async=True)
+
+# Kill a task that is running in the API.
+task.kill()
+```
+
+Or if you
+
+```python
+
+task = scenario.simulate(..., run_async=True)
+
+# Get the current status of the task.
+status = task.get_status()
+
+# Block until the results are ready and get an output object.
+# `output` objects vary depending on the scenario, exposing methods to
+# manipulate the results that are relevant for that scenario!
+output = task.get_output()
+
+# Alternatively, download the raw output files!
+
+# If you don't know the files generated by your task, you can get an object
+# representing information on the output files.
+outputs_info = task.get_output_files_info()
```

Comment:
why not just "get_output_info()"
maybe you can give more info than just the files.

Reply:
because this one is the same for all tasks, giving more related information other than the files would be scenario/simulator dependent. There's already a `get_output()` method that retrieves outputs in a "scenario specific" way